src/Scene/Scene.js

```diff
@@ -75,6 +75,9 @@ define('Scene/Scene', [
 
         this.gLDebug = gLDebug;
         this.gfxEngine = c3DEngine(this,positionCamera,viewerDiv, debugMode,gLDebug);
+        this.needsUpdate = true;
+        this.forceUpdate = true;
```

Comment:
The two boolean have different meaning: needsUpdate indicates that the scene is not in a stable state (still needs subdivision or textures...), forceUpdate means the scene has to be rendered, even if the scene has not changed.
The scene can need an update but not update for performance reasons: typically, we want to wait for a certain amount of commands to be executed before rendering and updating the scene.

I clarified all this in a new commit.